pkg/controller/v1alpha1/trainedmodel/controller_test.go

```diff
@@ -370,6 +501,18 @@ var _ = Describe("v1beta1 TrainedModel controller", func() {
 			}
 			Expect(k8sClient.Create(ctx, isvc)).Should(Succeed())
 
+			inferenceService := &v1beta1.InferenceService{}
+			Eventually(func() bool {
+				err := k8sClient.Get(ctx, serviceKey, inferenceService)
+				if err != nil {
+					return false
+				}
+				return true
+			}, timeout, interval).Should(BeTrue())
+
+			inferenceService.Status.Status = statusConditions
+			Expect(k8sClient.Status().Update(context.TODO(), inferenceService)).To(BeNil())
```

Comment:
Add test to check the trained model status condition?

Reply:
Added it here: https://github.com/kubeflow/kfserving/pull/1402/files#diff-91393e839bab87f65a6a1b532750bb1b0753ca29cf828612b9d8ac8cdf2ba7c1R428